lib/compat/wordpress-6.5/fonts/class-wp-font-collection.php

```diff
@@ -162,7 +162,7 @@ private function load_from_url( $url ) {
 
 				$data = json_decode( wp_remote_retrieve_body( $response ), true );
 				if ( empty( $data ) ) {
-					return new WP_Error( 'font_collection_decode_error', __( 'Error decoding the font collection data from the REST response JSON.', 'gutenberg' ) );
+					return new WP_Error( 'font_collection_decode_error', __( 'Error decoding the font collection data from the http response JSON.', 'gutenberg' ) );
```

Comment:
I'd assume HTTP to be capitalized here.

Reply:
Updated here: https://github.com/WordPress/gutenberg/pull/58675/commits/59e6fdee0a5ce8009913cc54a5f7c22703a3ef63